fe/fe-core/src/main/java/org/apache/doris/alter/RollupJobV2.java

```diff
@@ -134,6 +134,10 @@ public class RollupJobV2 extends AlterJobV2 implements GsonPostProcessable {
     @SerializedName(value = "storageFormat")
     private TStorageFormat storageFormat = TStorageFormat.DEFAULT;
 
+    // used for delete decommission tablet
+    @SerializedName(value = "watermarkTxnId")
+    private long watermarkTxnId = -1;
+
```

Comment:
Name of watermarkTxnId may be too general, could use `tabletDeleteWatermarkTxnId`

Reply:
done